OracleWebLogic/samples/12212-domain/container-scripts/createServer.sh

```diff
@@ -2,11 +2,24 @@
 #
 # Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
 #
+export DOMAIN_HOME=/u01/oracle/user_projects/domains/$DOMAIN_NAME
+echo "Domain Home: " $DOMAIN_HOME
+
+if [ -z $ADMIN_PASSWORD ]; then
+   echo "      ----> NO 'weblogic' admin password set"
+   echo ""
+else
+   s=${ADMIN_PASSWORD}
+   echo "      ----> 'weblogic' admin password: $s"
+fi
+sed -i -e "s|ADMIN_PASSWORD|$s|g" /u01/oracle/commonfuncs.py
```

Comment:
Oh, I see now... So `ADMIN_PASSWORD` will be replaced with the actual password. Maybe it would be better if the password was stored in a file, and then read from that file in the WLST scripts instead of using `sed` to replace a hardcoded string. Otherwise it is hard to understand/debug what's going on